clash-ghc/src-bin-841/Clash/GHCi/UI.hs

```diff
@@ -625,30 +625,36 @@ runGHCi paths maybe_exprs = do
   case maybe_exprs of
         Nothing ->
           do
+            -- Set different defaulting rules (See #280)
+            runGHCiExpressions ["default ((),[],Integer,Int,Double,String)"]
```

Comment:
Any reason to prefer `Integer` over `Int`? Should we ever infer `Integer`s anyway?

Reply:
Yes, otherwise you get weird overflow behavior when you type big literals into the prompt:
```
GHCi, version 8.8.1: https://www.haskell.org/ghc/  :? for help
Loaded package environment from /home/christiaan/devel/clash-compiler/.ghc.environment.x86_64-linux-8.8.1
Prelude> default (Int,Integer)
Prelude> 4444444444444444444444444444444444444444444444444444444444444444444444444444
-4099276460824344804
```